react-effects/my-app/src/List.tsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+import { useEffect, useState } from 'react';
+import { readItems, type Item } from './read';
+
+export function List() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [items, setItems] = useState<Item[]>([]);
+  const [error, setError] = useState<unknown>();
+
+  useEffect(() => {
+    async function effecting() {
+      try {
+        const settlingLisMapThrough = await readItems();
+
+        setItems(settlingLisMapThrough);
+      } catch (err) {
+        console.log('Error message:', err);
+        console.log('Cannot get:', setIsLoading(!isLoading));
```

Comment:
It's a bad idea to call an important function like setIsLoading inside of a console.log. Usually you delete all the console.log statements when you are done with the code. If you did that here, you would delete the important call to setIsLoading.